ofrak_core/ofrak/cli/command/gui.py

```diff
@@ -36,9 +36,26 @@ def create_parser(self, ofrak_subparsers):
             action="store_true",
             help="Don't open the browser to the OFRAK GUI",
         )
+        gui_parser.add_argument(
+            "--file",
+            "-f",
+            required="False",
+            action="append",
+            help="Path to a file to load into OFRAK when starting the GUI (multiple may be "
+            "provided)",
+            default=[],
+        )
         self.add_ofrak_arguments(gui_parser)
         return gui_parser
 
     async def ofrak_func(self, ofrak_context: OFRAKContext, args: Namespace):  # pragma: no cover
-        server = await open_gui(args.hostname, args.port, open_in_browser=(not args.no_browser))
+        if len(args.file):
+            for path in args.file:
+                _ = await ofrak_context.create_root_resource_from_file(path)
+        server = await open_gui(
+            args.hostname,
+            args.port,
+            open_in_browser=(not args.no_browser),
+            ofrak_context=ofrak_context,
+        )
```

Comment:
Interesting. Did you consider having `ofrak gui -f` take only one file, and then open that file (the way identify does)?

That feels more intuitive to me, because it remove one step (having to click on a specific resource in the GUI when it loads). I'd be in favor of allowing one file argument.

Reply:
To close the loop on this: we talked offline, and the conclusion was that `-f` should open the GUI to the resource if just one file is loaded, and open to the start view if multiple files are passed.